### DISCUSSION TITLE
About the bug of tool image-resizer

### DISCUSSION THREAD

**Discussion by feixueck** [he3-app/he3-feedback#289] [Bug Report]
### Version
image-resizer 1.1.3

### Environment
Windows v1.3.23

### What is actually happening?

不勾选: 保持纵横比 (比如 512X512)
保存之后: 仍然是等比例缩放 . (281X512)

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/291
